tools/inference.py

```diff
@@ -388,6 +388,10 @@ def run(
             )  # 1
 
             ts = time.time()
+            # Convert float64 to float32 to satisfy tensorflow requirements
+            features[
+                features.select_dtypes(np.float64).columns
+            ] = features.select_dtypes(np.float64).astype(np.float32)
```

Comment:
Should 64 vs 32 be configurable?

Reply:
That makes sense, just pushed a new commit with the change.